### PR TITLE
Feature build project schema

### PR DESCRIPTION
**Pull request by tanveerisonline**
In this feature branch i performed the following requirements:

🟡###  Project requirements

- [ ] Create a new app. Give it a meaningful name.
- [ ] Initialize your project with Git.
- [ ] Make sure that your project has Postgres database set up.
- [ ] -Build your project schema.

**Create and run the necessary migration files.**

- [ ] **Table and column names should match the ERD diagram. note: photo for the user's table should be a link to an image**

- [ ] **Foreign keys should be included.**

- [ ] **Pay attention that in the posts table the column holding the foreign key to the user's table should be named author_id**

- [ ] **All columns that are foreign keys should have a corresponding index.**

### CODE REVIEW

File: db/migrate/20230328180336_add_foreign_key_to_likes.rb [hunk 1]
```diff
@@ -0,0 +1,8 @@
+class AddForeignKeyToLikes < ActiveRecord::Migration[7.0]
+  def change
+  add_foreign_key :likes, :posts, column: :post_id
+  add_foreign_key :likes, :users, column: :user_id
```
Comment (AnselemOdims) [inline db/migrate/20230328180336_add_foreign_key_to_likes.rb]: - [ ] Please kindly ensure that the foreign  key references the `author_id` as specified in the schema diagram

File: db/migrate/20230328181907_add_index_to_comments.rb [hunk 2]
```diff
@@ -0,0 +1,6 @@
+class AddIndexToComments < ActiveRecord::Migration[7.0]
+  def change
+  add_foreign_key :comments, :users, column: :user_id
```
Comment (AnselemOdims) [inline db/migrate/20230328181907_add_index_to_comments.rb]: - [ ] Please kindly ensure that the foreign  key references the `author_id` as specified in the schema diagram

**Review by AnselemOdims — CHANGES_REQUESTED** [review]
### Hi @tanveerisonline , 
You have been doing a tremendous job so far :ok_hand:
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
PR has a title and summary. :heavy_check_mark:
No linter errors :heavy_check_mark:

## Required Changes ♻️
- [ ] Please also check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity! You can tag me using @AnselemOdims for a quicker response

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by algerina — APPROVED** [review]
Hi @tanveerisonline ,

Great job so far! 

May I highlight the following: 
- [x] The project is set correctly✔️ 
- [x] Well done implementing the previous reviewer's requirements✔️ 

## STATUS: APPROVED :muscle::clap:

Congratulations! :tada:

Your project is complete!  It's time to merge it :shipit:



 
 ## Optional suggestions

 - Please check the comment under the review.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread and tag me @algerina if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)
___
_If anything seems to be unclear to you, reach out and ask anything by commenting on this Pull Request and mention me as [Algerina](https://github.com/algerina), I will receive a notification and will respond._

[GitHub](https://github.com/algerina) | [LinkedIn](https://linkedin.com/in/amel-khiri) | [Twitter](https://twitter.com/AmelQahwadji)

_Reviewed by [Amel Khiri ](https://github.com/algerina)_

**Pull request merged**